x/swap/types/pair.go

```diff
@@ -0,0 +1,94 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Pair defines a tradable token pair
+type Pair struct {
+	TokenA    string  `json:"token_a" yaml:"token_a"`
+	TokenB    string  `json:"token_b" yaml:"token_b"`
+	RewardAPY sdk.Dec `json:"reward_apy" yaml:"reward_apy"`
```

Comment:
Swap module shouldn't have any knowledge of rewards, so this can be removed. This will be defined in `incentive`.